File: cFizzBuzz/fb.c
#include <stdio.h>

int main()
{
    int i;

    for (i = 1; i <= 100; i++) {
        if (!(i % 3))
            printf("Fizz");
        if (!(i % 5))
            printf("Buzz");
        if (!(i % 3) || !(i % 5)) {
            printf("\n");
            continue;
        }
        printf("%i\n", i);
    }

    return 0;
}

File: stuff/fb.go
package main

import (
    "fmt"
    "math"
)

/*
 * ITC: Horrible implementation of FizzBuzz in Go
 * If this is anywhere near an optimal solution, go is already dead
 */

func main() {
    for i := 1; i <= 100; i++ {
        if math.Mod(float64(i), 3) == 0 && math.Mod(float64(i), 5) == 0 {
            fmt.Println("FizzBuzz")
            continue
        }
        if math.Mod(float64(i), 3) == 0 {
            fmt.Println("Fizz")
            continue
        }
        if math.Mod(float64(i), 5) == 0 {
            fmt.Println("Buzz")
            continue
        }
        fmt.Println(i)
    }
}

